input/pagecontent/2-3-Events.md

```diff
@@ -106,7 +106,7 @@ SelectionEventName ::= ( FHIRresource  ) '-' ( 'select' )
 
 {% include img.html img="SelectionEventName.png" caption="Figure: Selection Event-name specification" %}
 
-The `FHIRresource` indicates the context of the selection. The `context` element in a select event typically contains two fields. One with the name of the `FHIRresource` holding the anchor resource and one or more named `select` indicating the resources that are selected.
+The `FHIRresource` indicates the context of the selection. The `context` element in a select event typically contains two fields. One field with the name of the fhir-resource holding the anchor resource, and one or more fields named select indicating the resources that is selected.

```

Comment:
Hence my text proposal would be:

The `FHIRresource` indicates the context of the selection. The `context` element in a select event contains two fields. One field with the name of the FHIR resource which is the anchor context, and one field named `select` which is an array indicating the resources that are selected or an empty array indicating that no resource is selected.

Reply:
I made the empty array phrase into it's own sentence, but otherwise took all of the above. Thank you for paying attention and catching this!